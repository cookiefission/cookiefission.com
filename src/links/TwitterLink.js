import React from 'react';
import IconLink from './IconLink';

const TwitterLink = () => {
  return(
    <IconLink href='https://twitter.com/cookiefission' iconName='twitter' />
  )
};

export default TwitterLink;
