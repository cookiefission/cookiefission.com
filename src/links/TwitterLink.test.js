import React from 'react';
import TwitterLink from './TwitterLink';
import renderer from 'react-test-renderer';

describe('<TwitterLink />', () => {
  it('matches the snapshot', () => {
    const component = renderer.create(
      <TwitterLink />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
