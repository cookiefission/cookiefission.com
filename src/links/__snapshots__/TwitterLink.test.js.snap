exports[`<TwitterLink /> matches the snapshot 1`] = `
<a
  href="https://twitter.com/cookiefission"
  target="blank">
  <div
    className="Icon-wrapper">
    <i
      className="fa fa-twitter fa-lg" />
  </div>
</a>
`;
